### DISCUSSION TITLE
MySQL preview

### DISCUSSION THREAD

**Discussion by paulm17** [hasura/graphql-engine#7912] [General] [answered]
I'm about to fire up hasura with docker and postgres to use with blitzjs and relay.  Excited to know more about this space.

My main question is when should we expect MySQL to be general availability?  And for me, more importantly.  How likely would singlestore interoperability be?  Or as a minimum just connection, with read and write access?  Side stepping other features as a minimum...

https://www.singlestore.com/

Thanks!

**Comment by codek** [comment, marked as answer]
its very weird, tonnes of work done on the mysql preview and it's all ground to a halt.

**Reply by paulm17** [reply]
Totally forgot about this.  😃 

Migrated over to Dgraph.  Gonna mark this as the answer and unsub.

**Reply by codek** [reply]
thanks for letting me know! hadnt come across that one, pretty interesting. good luck!
